astropy/cosmology/flrw/tests/test_w0wzcdm.py

```diff
@@ -178,3 +230,13 @@ def test_de_densityscale():
         cosmo.de_density_scale([1.0, 2.0, 3.0]),
         rtol=1e-7,
     )
+
+    # Flat tests
+    cosmo = w0wzCDM(H0=70, Om0=0.3, Ode0=0.7, w0=-1, wz=0.5)
+    flatcosmo = Flatw0wzCDM(H0=70, Om0=0.3, w0=-1, wz=0.5)
+
+    assert u.allclose(
+        cosmo.de_density_scale(z),
+        flatcosmo.de_density_scale(z),
+        rtol=1e-4,
```

Comment:
Doesn't this fit in a single line?
```suggestion
        cosmo.de_density_scale(z), flatcosmo.de_density_scale(z), rtol=1e-4
```

Reply:
✅